airflow/www/views.py

```diff
@@ -5330,6 +5330,14 @@ def _calculate_graph(self):
 
     @staticmethod
     def _node_dict(node_id, label, node_class):
+        def trunc_uri(val):
+            if len(val) > 25:
+                return val[0:23] + '...'
```

Comment:
Is there somewhere that we pass the full URI so we can show it in a tooltip?

Reply:
Yes, it is the node I'd, but I'm not sure if that makes it into dom. I tried looking but didn't see?

Reply:
As per another conversation, let's just show the whole URI for now so a user has the whole context and worry about shortening it later.

Reply:
ok, sure

Reply:
yeah it seems we don't have tooltip code in the dag dependencies js